src/Extensions/IntrospectionProvider.php

```diff
@@ -22,7 +23,16 @@ class IntrospectionProvider extends Extension
      */
     public function types(HTTPRequest $request)
     {
-        $manager = $this->owner->getManager();
+        try {
+            $manager = $this->owner->getManager();
+        } catch (Exception $ex) {
+            if ($ex->getMessage() == "Authentication required") {
+                return (new HTTPResponse(json_encode(["error" => "Authentication required"]), 500))
```

Comment:
500 isn't the right response code here.
```suggestion
                return (new HTTPResponse(json_encode(["error" => "Authentication required"]), 401))
```